Update glossary.md (#2074)
dfinity/portal: docs/references/glossary.md

#### dapp

A **dapp**, or decentralised application, is a software program that runs
on a decentralised computer network instead of a single computer. On the
[Internet Computer](#internet-computer-ic) dapps are composed of
[canister](#canister) smart contracts.

#### data center

A **data center** (DC) is a physical site that hosts
[nodes](#node) which contribute to the [Internet
Computer](#internet-computer-ic). It includes the hardware and
software infrastructure required for node deployment. 

> Data centers are node are selected and vetted by the [NNS](#NNS).

Reviewer: ```suggestion
Data centers are nodes that are selected and vetted by the [NNS](#NNS).
```

> Data centers are node are selected and vetted by the [NNS](#NNS).

Author: Are there other aspects of data centers we want to highlight, @dprats ?

#### node

> A **node** is a physical hardware devices run by independent

Reviewer: ```suggestion
A **node** is a physical hardware device run by independent
```

A **node operator** (NO) is a non-canister
[principal](#principal) who has the authority to add/remove
[nodes](#node) to/from the

> [IC](#internet-computer-ic-ic), delegated by 

Author: ```suggestion
[IC](#internet-computer-ic-ic), by delegation of the
```
